Build instructions in br‘s readme.md is outdated

## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

build br component as https://github.com/pingcap/tidb/blob/master/br/README.md said

### 2. What did you expect to see? (Required)

successful build after following instructions

### 3. What did you see instead (Required)

root@dev:~/tidb/br# make
make: *** No targets specified and no makefile found.  Stop.

### 4. What is your TiDB version? (Required)

latest

<!-- Paste the output of SELECT tidb_version() -->

--- a/br/CONTRIBUTING.md
+++ b/br/CONTRIBUTING.md
@@ -17,11 +17,19 @@ Developing BR requires:
 * [Go 1.16+](http://golang.org/doc/code.html)
 * An internet connection to download the dependencies
 
-Simply run `make` to build the program.
+1. Navigate to the tidb directory
 
 ```sh
-make
+cd ../tidb
 ```
+
+2. run `make build_br` to build the program.
+
+```sh
+make build_br
+```
+
+After all these, you will find `br` in `tidb/bin` directory
 
 ### Running tests
 

--- a/br/README.md
+++ b/br/README.md
@@ -30,7 +30,7 @@
 To build binary and run test:
 
 ```bash
-$cd ../tidb
+$ cd ../tidb
 $ make build_br
 $ make test
 ```

--- a/br/README.md
+++ b/br/README.md
@@ -30,7 +30,8 @@
 To build binary and run test:
 
 ```bash
-$ make
+$cd ../tidb
+$ make build_br
 $ make test
 ```
 
@@ -100,7 +101,7 @@ tiup bench tpcc --warehouses 1 prepare
 mysql --host 127.0.0.1 --port 4000 -E -e "SELECT COUNT(*) FROM test.order_line" -u root -p
 
 # Build br.
-make build
+make build_br
 
 # Backup TPC-C test data.
 bin/br backup table --db test \

